src/measure_theory/measure_space.lean

```diff
@@ -996,6 +1000,16 @@ begin
     outer_measure.restrict_apply]
 end
 
+/-- Alternate version of `measure.restrict_apply`.
+  Requires that `s` is measurable instead of `t`. -/
```

Comment:
I think that docstrings are supposed to be self-contained, i.e. not dependent on any other lemmas for context.

Having said that, `measure.restrict_apply` itself does not currently have a docstring. Perhaps it is worthwhile to add one (you can then reference the existence of the other theorem in each of the new docstrings).

@sgouezel will have a better idea about this than I.

Reply:
Yes, docstrings should be self-contained. Ideally, you would keep what you already have (the reference to `measure.restrict_apply` is definitely useful here), but you would add a description of the mathematical content of the lemma.